vagrant/puppet/jackrabbit/manifests/init.pp

```diff
@@ -0,0 +1,26 @@
+class jackrabbit(
```

Comment:
the actual reason of this module was to separate the jackrabbit-specific code, but now I've solved the jackrabbit issues using the shell provisioner and that's just something like an initializer (comparable to the behavior of the *cmf::infrastructure::phpcr::mysql_wrapper*, so it should be moved there.